public/data/AnanjayGurjar.json

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
There's an extra line here.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix